braindecode/datasets/ecog_bci_competition.py

```diff
@@ -0,0 +1,113 @@
+import mne
+import numpy as np
+import scipy.io
+
+from braindecode.datasets import BaseDataset, BaseConcatDataset
+
+
+class EcogBCICompetition4(BaseConcatDataset):
+    """BCI competition IV ECoG dataset with finger movements.
+
+    Contains ECoG recordings for three patients moving fingers during the experiment.
+    Targets correspond to the time courses of the flexion of each of five fingers.
+    See http://www.bbci.de/competition/iv/desc_4.pdf for the dataset description.
+
+    Test labels can be downloaded from:
+    http://www.bbci.de/competition/iv/results/ds4/true_labels.zip
+
+    Data can be downloaded from (Dataset 4):
+    http://www.bbci.de/competition/iv/#dataset4
```

Comment:
these data are super old. I am sure we could now make them available without password eg storing them on OSF.

I would contact the authors of the competition to ask for permission.

Maybe @cbrnr nows who to ask this.

I really don't like that an example does not work out of the box. And you won't be able to test it in CI or to build any doc. It's for me a blocker.

Reply:
Not sure who "owns" the data, but if it's BBCI I'd contact Benjamin Blankertz. If not, either Gerwin Schalk or Kai J. Miller are listed. I know Benjamin and Gerwin, if you want I could ask them.